docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
       "serve",
       "--host=0.0.0.0",
       "--port=5555",
-      "--ca googleca"
+      "--ca googleca",
```

Comment:
it also needs an equals sign in between ca and googleca

Reply:
yep 😅